buggy_script.py

```
@@ -0,0 +1,48 @@
+
+    # 6. 语法错误：在 if 条件中使用了赋值运算符 (=) 而不是比较运算符 (==)
+    user_input = "yes"
+    if user_input = "yes":
+        print("User agreed.")
+
+    # 7. 运行时错误：IndexError (索引越界)
+    items = ["Apple", "Banana", "Orange"]
+    for i in range(len(items) + 1):
+        print(f"Item {i}: {items[i]}")
+
+    # 8. 运行时错误：ZeroDivisionError (除以零)
```

Ruff (0.14.10)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)